packages/Algernon.package/ALGResultItemMorph.class/class/actionFontSize.st

```diff
@@ -1,3 +1,3 @@
 constants
 actionFontSize
-	^ 13
\ No newline at end of file
+	^ 8
```

Comment:
I complained it was to small, so you made it smaller?

Reply:
I also changed the Font-Size adaption function (`TextStyle default fontOfPointSize`) to use points instead of "height".
But there seems to be some "voodoo" involved, because there seem to be fixed font sizes, so I can't scale them freely.